index.html

```diff
@@ -1,348 +1,328 @@
-                                <h3>Gain + Glory</h3>
-                            </div>
-                            <ul class="technologies">
-                                <li>Javascript</li>
-                                <li>Ruby on rails</li>
-                                <li>HTML</li>
-                                <li>CSS</li>
-                            </ul>
-                        </div>
-                        <div class="green-btn">
-                            <a href="single-project.html">See Project</a>
-                        </div>
+                    <div class="green-btn">
+                        <a href="single-project.html">See Project</a>
                     </div>
-
                 </div>
 
-            </section>
-            <section class="section">
-                <div class="about-bg-imgs">
-                    <img src="assets/images/about-me/about-me-left.svg" alt="bg-image"/>
-                    <img src="assets/images/about-me/aboutmedesktop-right.svg" alt="bg-image" />
-               </div>
-                <div class="about-header">
-                    <div class="title">
-                        <h2 class="primary-text">About Me</h2>
-                        <hr class="line" />
+                <div class="grid-item">
+                    <div class="item-image">
+                        <!-- <img src="" alt="place image here" /> -->
+                    </div>
+                    <div class="item-description">
+                        <div class="item-title">
+                            <h2>Mult-Post Stories</h2>
+                            <h3>Gain + Glory</h3>
+                        </div>
+                        <ul class="technologies">
+                            <li>Javascript</li>
+                            <li>Ruby on rails</li>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
                     </div>
-                    <p class="about-text">
-                        I can help you build a product , feature or website Look
-                        through some of my work and experience! If you like what
-                        you see and have a project you need coded,
-                        don’t hestiate to contact me.
-                    </p>
                     <div class="green-btn">
-                        <a href="">Get My Resume</a>
+                        <a href="single-project.html">See Project</a>
                     </div>
                 </div>
-                <ul class="grid-container skills-set">
-                    <li class="grid-item skill-item">
-                        <div class="item-icon">
-                            <img src="assets/images/lang-icon.svg" alt="icon" />
+
+                <div class="grid-item">
+                    <div class="item-image">
+                        <!-- <img src="" alt="place image here" /> -->
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean. Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.